defcon/plugins/dayshours.py

```diff
@@ -0,0 +1,65 @@
+"""DefCon Static plugin."""
+import datetime
+from defcon.plugins import base
+
+
+class DaysHoursPlugin(base.Plugin):
+    """DefCon Dayshours plugin.
+
+    Config:
+
+    ```python
+    {
+      'days': [0,1,2,3,4], // List of Int to specify day rule : if x is in days (Monday being 0 and Sunday being 6)
+      'hours': [9,12], // Tuple of Int to specify hours rule : hours[0] < x < hours[1]
+      'defcon' : 1, // Int to use as defcon
```

Comment:
could be interesting to make it work with https://pypi.python.org/pypi/workalendar or https://pypi.python.org/pypi/businesstime, you'd get working days/times automatically configured and some bonuses